### PR TITLE
[EOBS-2359] Shift Coordinator has ability to edit locations

### PR DESCRIPTION
**Pull request by moderatemisbehaviour**
Removed the shift coordinators edit permissions but the 'form open' button was still appearing. Set the `no_open` option on the widget to `true` and this removed the button.

---

Before this pull request can be merged the following must be true.
- [ ] Unit tests pass (Travis integration).
- [ ] No code quality issues (Codacy integration).
- [ ] Approval from at least one developer and at least one tester.

If you are a BJSS contributor there are some additional conditions.
- [ ] All client module unit tests pass.
- [ ] The *Pull Request* field in the work tab of the JIRA issue is populated.
- [ ] The JIRA issue contains a description of the root cause and the solution in the comments.
- [ ] There are no bugs against the JIRA issues in the current sprint.

### CODE REVIEW

File: nh_clinical/security/ir.model.access.csv [hunk 1]
```diff
@@ -22,7 +22,7 @@ base_group_access_nh_clinical_user_responsibility_allocation_access,admin:access
 base_group_access_nh_clinical_user_management,admin:access_nh_clinical_user_management,model_nh_clinical_user_management,nh_clinical.group_nhc_base,1,0,0,0
 ,,,,,,,
 admin_group_access_nh_clinical_pos,admin:access_nh_clinical_pos,model_nh_clinical_pos,nh_clinical.group_nhc_admin,1,1,1,0
-admin_group_access_nh_clinical_location,admin:access_nh_clinical_location,model_nh_clinical_location,nh_clinical.group_nhc_admin,1,1,1,0
+admin_group_access_nh_clinical_location,admin:access_nh_clinical_location,model_nh_clinical_location,nh_clinical.group_nhc_admin,1,1,1,1
```
Comment (Nayira) [inline nh_clinical/security/ir.model.access.csv]: This is out of scope for this issue. Please raise ticket to deal with it at a later date, once it's properly defined.

File: nh_clinical/security/ir.model.access.csv [hunk 2]
```diff
@@ -43,7 +43,7 @@ admin_group_access_nh_clinical_user_responsibility_allocation_access,admin:acces
 admin_group_access_nh_clinical_user_management,admin:access_nh_clinical_user_management,model_nh_clinical_user_management,nh_clinical.group_nhc_admin,1,1,1,0
 ,,,,,,,
 dev_group_access_nh_clinical_pos,admin:access_nh_clinical_pos,model_nh_clinical_pos,nh_clinical.group_nhc_dev,1,1,1,1
-dev_group_access_nh_clinical_location,admin:access_nh_clinical_location,model_nh_clinical_location,nh_clinical.group_nhc_dev,1,1,1,1
+dev_group_access_nh_clinical_location,admin:access_nh_clinical_location,model_nh_clinical_location,nh_clinical.group_nhc_dev,1,1,1,0
```
Comment (Nayira) [inline nh_clinical/security/ir.model.access.csv]: This is out of scope for this issue. Please raise ticket to deal with it at a later date, once it's properly defined.

File: nh_clinical/security/ir.model.access.csv (hunk 1 above)
Comment (moderatemisbehaviour) [reply nh_clinical/security/ir.model.access.csv]: Raised question [EOBS-400 What should permissions for the location model be?](https://jupiter.bjss.com/browse/EOBS-2400).

File: nh_clinical/security/ir.model.access.csv (hunk 2 above)
Comment (moderatemisbehaviour) [reply nh_clinical/security/ir.model.access.csv]: Raised question [EOBS-400 What should permissions for the location model be?](https://jupiter.bjss.com/browse/EOBS-2400).

**Pull request merged**